go.mod

```diff
@@ -287,6 +290,7 @@ replace (
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
 	github.com/gophercloud/gophercloud => github.com/gophercloud/gophercloud v0.1.0
 	github.com/gopherjs/gopherjs => github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1
+	github.com/gorilla/context => github.com/gorilla/context v1.1.1
```

Comment:
[1] i have no explanation where these are coming from.

the update process was:
```
make clean
./hack/pin-dependency.sh k8s.io/system-validators v1.4.0
./hack/update-vendor.sh
```

```
$ go version
go version go1.16 linux/amd64
```

Reply:
this is the library go.mod/sum

https://github.com/kubernetes/system-validators/blob/master/go.mod
https://github.com/kubernetes/system-validators/blob/master/go.sum

which haven't changed since v1.1.0
https://github.com/kubernetes/system-validators/commit/e32378eeef1a69890bc966f975208f0171936aac#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6

Reply:
cc @kubernetes/dep-approvers

Reply:
https://github.com/kubernetes/kubernetes/pull/98977#issuecomment-789167596